acceptance/selftest/record_cloud/volume-io/out.requests.txt

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
nit: it's hard to see what request belongs to which call, a more readable approach is to print out.requests.txt each time and delete it, see print_requests function in other tests: https://github.com/databricks/cli/blob/e1a8daa5fe08d3d6c851d66834fa5ea4aaa4edb5/acceptance/bundle/resources/apps/output.txt

Reply:
Good recommendation, updated the tests.

Reply:
Good recommendation, updated the tests.